app/forms/depositor_form.rb

```diff
@@ -1,6 +1,5 @@
 # frozen_string_literal: true
 
 # Form for collection depositors
-class DepositorForm < ApplicationForm
-  attribute :sunetid, :string
+class DepositorForm < ParticipantForm
```

Comment:
Since there is more than one attribute, having all participant forms subclass a common form.